### DISCUSSION TITLE
NetScaler ingress controller version 2.0.6 is not updated in Artifact hub

### DISCUSSION THREAD

**Discussion by subashd** [artifacthub/hub#4000] [General]
HI Team,
We have released NSIC version 2.0.6 helm chart today, but it is not getting updated in the artifacthub.
Could you please help us on this ?

gh-pages: https://github.com/netscaler/netscaler-helm-charts/tree/gh-pages
index.yaml: https://github.com/netscaler/netscaler-helm-charts/blob/gh-pages/index.yaml
Helm Repo: https://github.com/netscaler/netscaler-helm-charts/

**Comment by tegioz** [comment]
Hi @subashd 👋 

The index used by Artifact Hub (https://netscaler.github.io/netscaler-helm-charts/index.yaml) doesn't seem to have the same content as the one you shared (https://raw.githubusercontent.com/netscaler/netscaler-helm-charts/gh-pages/index.yaml).

```
wget -q -O index1.yaml https://netscaler.github.io/netscaler-helm-charts/index.yaml
wget -q -O index2.yaml https://raw.githubusercontent.com/netscaler/netscaler-helm-charts/gh-pages/index.yaml
diff index1.yaml index2.yaml
```

Artifact Hub uses the former (as configured by the publisher), which doesn't contain yet the new version.

Hope this helps 🙂

**Reply by subashd** [reply]
Yes it was not updated. Seems there was change in the way a site is hosted in GitHub. We have fixed the issue in GitHub and [this file](https://netscaler.github.io/netscaler-helm-charts/index.yaml)) is updated.

**Reply by subashd** [reply]
hi @tegioz 
still the repo in the artifacthub is not updated with the new version.

**Reply by subashd** [reply]
Finally, the latest version is updated. Thank you :)
